judge/migrations/0001_squashed_0084_contest_formats.py

```diff
@@ -0,0 +1,765 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-04-18 02:06
+from __future__ import unicode_literals
```

Comment:
Yikes

Reply:
What's so bad about this line? It's Django-generated.